src/components/OSCALControlProse.js

```diff
@@ -331,6 +406,39 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
           );
         })}
       {props.modificationDisplay}
+      {props.isEditable ? (
+        <>
+          <IconButton
+            onClick={(event) => {
+              setAnchorEl(event.currentTarget);
+            }}
+          >
+            <EditIcon fontSize="small" />
+          </IconButton>
+          <Grid container>
+            <OSCALControlPartEditor
+              anchorEl={anchorEl}
+              setAnchorEl={setAnchorEl}
+              controlId={props.controlId}
+              isUserDefinedImplementation={props.prose.match(
+                prosePlaceholderRegexpString
+              )}
+              onCancel={() => {
+                setAnchorEl(null);
+              }}
+              onFieldSave={(descriptionReference, implementationReference) => {
```

Comment:
Again, we shouldn't import functions like `onFieldSaveByComponentParameterValue` from a component then send them back in to rendering declaration of that component.